server/src/completion.ts

```diff
@@ -177,8 +177,12 @@ export interface CompletionAt {
     _originalText: string
 }
 
+export function isValidLocation(at : CompletionAt): boolean {
+    return at.file.length > 0 && at.line !== -1 && at._uri.length > 0 && !isRangeZeroEmpty(at._range)
```

Comment:
line info and uri can be changed on parsing and only _uri and _range should be used for check